pkg/worker/verifier/producer_worker.go

```diff
@@ -108,7 +109,10 @@ func (pw *ProducerWorker) newRecord(producerId int, sequence int64) *kgo.Record
 		pw.Status.AbortedTransactionMessages += 1
 	}
 
-	payload := make([]byte, pw.config.messageSize)
+	payload := pw.config.valueGenerator.Generate()
```

Comment:
Not sure why `valueGenerator` wasn't used here before instead of an empty `make([]byte)`?

Reply:
The reason is `rand.Read()` will generate `[]byte` objects that are very likely not UTF-8 decodable.

We could roll our own `random()` function using a sub-set of the alphabet, but it may be worth just reverting this change back.